manifests/example-template.yml

```diff
@@ -6,13 +6,16 @@ metadata:
     description: "OCP kubevirt linux, template"
     tags: "kubevirt,ocp,template,linux"
   labels:
-    miq.github.io/kubvirt-os: rhel-7
+    miq.github.io/kubevirt-os: rhel-7
     miq.github.io/kubevirt-is-vm-template: true
 objects:
 - apiVersion: kubevirt.io/v1alpha1
   kind: OfflineVirtualMachine
   metadata:
     name: ${NAME}
+    selector:
```

Comment:
The selector will only be in the `VirtualMachinePreset`, i.,e.:
```
metadata:
  selector:
    matchLabels:
      flavor: large
```

A VirtualMachine (!) will then have a label:
```
metadata
  labels:
    flavor: large
```

Thus the preset will select all VMs which have that label.

Reply:
OK. I will remove those. Let's wait for support and implement it once available